style.css

```diff
@@ -0,0 +1,299 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: "poppins";
```

Comment:
It's best to provide some fallback fonts - as you haven't done this the browser defaults to an ugly Serif font if there's an issue loading Poppins or you forget to load it, ALso, the README did say  to put fonts in vars. I would probaly do a value of something like this  Poppins, Arial, sans-serif That defaults to poppins, if it can't find it, it uses Arial which is a font that's installed in most computers already, and in the unlikely event it can't find that, it falls back to any Sans Serif font. MDN goes into a lot more depth if it intrests you https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Fundamentals#fonts Also, the body should be Quicksand, Poppins is for headers according to the readme